dist/index.html

```diff
@@ -21,11 +21,17 @@ <h2>Recent scores</h2>
           <h2>Add your score</h2>
           <form action="#" class="form">
             <input type="text" placeholder="Your name" class="name" /><br />
-            <input type="tel" placeholder="Your score" class="score" /><br />
+            <input type="number" placeholder="Your score" class="score" /><br />
             <button type="submit" class="add">Submit</button>
           </form>
         </div>
```

Comment:
- [ ] Great work @christianbiring1, this add score section needs to be positioned next to the recent score below is a comparison with the given design.

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168416217-d1dcc8c9-61fe-4c93-8162-0842edaff20a.png) |  ![image](https://user-images.githubusercontent.com/61408860/168416228-ff5b085a-68d6-4383-adaa-1121f3766f08.png)

Reply:
Hey @danmainah! great for the review but just to let you know that my app is stylized but I don't know how you couldn't see that.
>It's like you didn't run install the dependencies locally and run `npm start` when you open it up on your computer
...

![Screenshot from 2022-05-14 10-08-47](https://user-images.githubusercontent.com/97100091/168417077-f776f1f1-0da6-4c4a-a0ad-11de10cda002.png)

Reply:
Hi @christianbiring1,

Your app is well styled and meets the requirements. Kudos and keep soaring higher 💯